CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added an importer for Citavi backup files, support ".ctv5bak" and ".ctv6bak" file formats. [#8322](https://github.com/JabRef/jabref/issues/8322)
 - We added a feature to drag selected entries and drop them to other opened inactive library tabs [koppor521](https://github.com/koppor/jabref/issues/521).
 - We added support for the [biblatex-apa](https://github.com/plk/biblatex-apa) legal entry types `Legislation`, `Legadminmaterial`, `Jurisdiction`, `Constitution` and `Legal` [#8931](https://github.com/JabRef/jabref/issues/8931)
+- We added tests Implementation of Multiple Condition Coverage (MCC) for isNumber method in [NumericFieldComparator class](https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/gui/util/comparator/NumericFieldComparator.java).
```

Comment:
Since there is no change visible to the user there shouldn't be a changelig entry.